lib/ansible/modules/network/cnos/cnos_vlan.py

```diff
@@ -223,66 +215,39 @@ def main():
             vlanArg5=dict(required=False),),
         supports_check_mode=False)
 
-    username = module.params['username']
-    password = module.params['password']
-    enablePassword = module.params['enablePassword']
-    vlanArg1 = module.params['vlanArg1']
-    vlanArg2 = module.params['vlanArg2']
-    vlanArg3 = module.params['vlanArg3']
-    vlanArg4 = module.params['vlanArg4']
-    vlanArg5 = module.params['vlanArg5']
     outputfile = module.params['outputfile']
-    hostIP = module.params['host']
-    deviceType = module.params['deviceType']
 
     output = ""
-    if not HAS_PARAMIKO:
-        module.fail_json(msg='paramiko is required for this module')
-
-    # Create instance of SSHClient object
-    remote_conn_pre = paramiko.SSHClient()
-
-    # Automatically add untrusted hosts (make sure okay for security policy in
-    # your environment)
-    remote_conn_pre.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-
-    # initiate SSH connection with the switch
-    remote_conn_pre.connect(hostIP, username=username, password=password)
-    time.sleep(2)
-
-    # Use invoke_shell to establish an 'interactive session'
-    remote_conn = remote_conn_pre.invoke_shell()
     time.sleep(2)
 
     # Enable and enter configure terminal then send command
-    output = output + cnos.waitForDeviceResponse("\n", ">", 2, remote_conn)
-
-    output = output + \
-        cnos.enterEnableModeForDevice(enablePassword, 3, remote_conn)
+    cmd = [{'command': '\n', 'prompt': None, 'answer': None}]
```

Comment:
I will try to handle this in terminal plugin.  There are few old Lenovo switches in which without pressing an ENTER, the '>' prompt wont come. If > prompt dont come, it waits and times out. So to avoid this, I have added this. May be irrelevant with latest switches.

Reply:
Removed. We need not care switches of 3 years back